src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -120,6 +120,7 @@ func (client *MobileClient) updateConfig() error {
 	var buf []byte
 	var err error
 	if buf, err = pullConfigFile(client.fronter); err != nil {
+		log.Fatalf("Could not update config: '%v'", err)
```

Comment:
Just created https://github.com/getlantern/firetweet/issues/86 and https://github.com/getlantern/lantern/pull/2554

Can you review https://github.com/getlantern/lantern/pull/2554 @fffw?